packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "httpaf-lwt-unix"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/httpaf"
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "faraday-lwt-unix"
+  "httpaf"
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.5